src/guiFormSpecMenu.cpp

```diff
@@ -2388,9 +2389,15 @@ void GUIFormSpecMenu::drawList(const ListDrawSpec &s, int phase,
 						utf8_to_wide(item.getDefinition(m_client->idef()).description);
 				else
 					tooltip_text = utf8_to_wide(desc);
-				// Show itemstring as fallback for easier debugging
-				if (!item.name.empty() && tooltip_text.empty())
-					tooltip_text = utf8_to_wide(item.name);
+
+				if (!item.name.empty()){
+					//Append itemstring if enabled in settings
+					if (tooltip_text.empty())
+						tooltip_text += utf8_to_wide(item.name);
+					if (m_tooltip_show_itemstring){
```

Comment:
this will show `default:stone [default:stone]`, not sure whether this is good or not. I guess it's rare enough

Reply:
I thought it was the most consistent